xml/System.Security.Cryptography.X509Certificates/X509CertificateLoader.xml

```diff
@@ -28,7 +28,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Provides methods for loading an X.509 certificate or a PKCS#12 PFX containing certificates.</summary>
+    <summary>Provides methods for loading an X.509 certificate or PKCS#12 PFX-containing certificates.</summary>
```

Comment:
"PKCS#12 PFX" is a singular noun, so, that's why I previously put "a" in front of it. That singular thing can contain multiple certificates.

Reply:
Ah got it. I guess this is why our style guide wants us to avoid -ing words, since they can be ambiguous.